app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container.rb

```diff
@@ -43,9 +70,9 @@ def raw_cloud_object_store_container_clear
 
   def self.raw_cloud_object_store_container_create(ext_management_system, options)
     # frontend stores name as :name, but amazon expects it as :bucket
-    options[:bucket] = options.delete(:name) unless options[:name].nil?
+    region = options.delete(:region)
 
-    region = options[:create_bucket_configuration][:location_constraint]
+    options[:create_bucket_configuration] = {:location_constraint => region}
     connection = ext_management_system.connect(:region => region)
     bucket = connection.create_bucket(options)
```

Comment:
@GilbertCherrie if `:bucket` and `:create_bucket_configuration` are the only two options you could do:
```
    connection = ext_management_system.connect(:region => options[:region])
    bucket = connection.create_bucket(:bucket => options[:name], :create_bucket_configuration => {:location_constraint => options[:region]})
```